app/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
     <!-- APIs and Integrations -->
     <string name="parse_app_id">traveltrails</string>
-    <string name="google_api_key">AIzaSyD4JhCWAAVr5oxzC9dzYQl6ed2OeHXuLdQ</string>
+    <string name="google_api_key">AIzaSyBy3aSmVTrzxJ3ycvpIJ26Jv2PYLQHFYSE</string>
```

Comment:
I think we should stick with one API key. I'm fine with using your key, but we should make sure everyone will have access to it. I think at some point (since we'll be using maps) we're going to need to do this: http://guides.codepath.com/android/Collaborating-on-Projects-with-Git#google-maps-access-across-computers